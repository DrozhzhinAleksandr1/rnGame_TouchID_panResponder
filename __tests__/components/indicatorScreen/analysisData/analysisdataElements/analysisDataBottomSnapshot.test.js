import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { AnalysisDataBottom } from '../../../../../app/components/indicatorScreen/analysisData/analysisDataElements/analysisDataBottom';

it('create snapshot AnalysisDataBottom', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<AnalysisDataBottom
        text1={'R1:'}
        number1={0.94765}
        text2={'R2:'}
        number2={0.94276}
        text3={'R3:'}
        number3={0.94965}
        discriptions={'Показатели:'}
        up={3}
        down={0} />);
    expect(toolbarSnapshot).toMatchSnapshot();
});
