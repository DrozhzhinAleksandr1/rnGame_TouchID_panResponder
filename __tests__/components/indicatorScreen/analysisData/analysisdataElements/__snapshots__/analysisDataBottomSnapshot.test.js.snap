// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot AnalysisDataBottom 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "borderTopColor": "#5A6772",
      "borderTopWidth": 0.25,
      "flex": 3,
      "flexDirection": "column",
      "justifyContent": "center",
      "minHeight": 35,
      "width": "90%",
    }
  }
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "center",
        "marginBottom": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Light",
          "fontSize": 11,
          "lineHeight": 13,
          "marginHorizontal": 5,
        }
      }
    >
      Показатели:
      :
    </Text>
    <Up />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Regular",
          "fontSize": 12,
          "lineHeight": 14,
          "marginHorizontal": 5,
        }
      }
    >
      3
    </Text>
    <Down />
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Regular",
          "fontSize": 12,
          "lineHeight": 14,
          "marginHorizontal": 5,
        }
      }
    >
      0
    </Text>
  </Component>
  <Component
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-around",
        "width": "100%",
      }
    }
  >
    <SlidingValue
      number={0.94765}
      text="R1:"
    />
    <SlidingValue
      number={0.94276}
      text="R2:"
    />
    <SlidingValue
      number={0.94965}
      text="R3:"
    />
  </Component>
</Component>
`;
